ag_data/simulator.py

```diff
@@ -151,27 +113,16 @@ def logLiveMeasurements(self, frequencyInHz, sleepTimer=0):
         does not have enough resources to generate required sensor measurements. Please
         notice that for some sensors, data generation use various random number generations
         extensively.
-
-        Arguments:
-
-            frequencyInHz {float} -- frequency for measurement generation. This method will
-            do its best to achieve the frequency at its best.
-
-        Keyword Arguments:
-
-            sleepTimer {float} -- time in seconds before automatically stop generate new
-            measurements. (default: {0} which will result in generating measurements
-            infinitely)
```

Comment:
I guess some of deleted comments contain useful information like this? e.g zero means generating measurements infinitely

Reply:
I can add thumbtack!

Reply:
I can add them back!

Reply:
any other deleted comments should be recovered?